src/main/java/com/parasoft/findings/bamboo/ReportCollector.java

```diff
@@ -77,6 +77,8 @@ public TestCollectionResult collect(File file) throws Exception {
 
     private InputStream getInputStream(File file, String xslFile)
             throws FileNotFoundException, TransformerException {
+        // Resolve the conflict of TransformerFactory and Xalan Dependency
```

Comment:
Maybe we can consider to use Saoxn. 
Saxon is a commonly used XSLT processor in the industry, with stronger performance and higher security.
Our plugin for other platforms use Saoxn as well.